socorro/external/es/crashstorage.py

```diff
@@ -522,92 +504,3 @@ def _submit_crash_to_elasticsearch(
                     exc_info=True,
                 )
                 raise
-
-
-class ESCrashStorageRedactedSave(ESCrashStorage):
-    required_config = Namespace()
-    required_config.namespace("es_redactor")
-    required_config.es_redactor.add_option(
-        name="redactor_class",
-        doc="the name of the class that implements a 'redact' method",
-        default="socorro.external.crashstorage_base.Redactor",
-        from_string_converter=class_converter,
-    )
-    required_config.es_redactor.add_option(
-        name="forbidden_keys",
-        doc="a list of keys not allowed in a redacted processed crash",
-        default=(
-            "json_dump, "
-            "upload_file_minidump_flash1.json_dump, "
-            "upload_file_minidump_flash2.json_dump, "
-            "upload_file_minidump_browser.json_dump"
-        ),
```

Comment:
This did have an effect on things, however, recent overhauls of how indexing works have pared down indexing to _only_ index fields listed in super search fields and these four are not in that list. Given that, the indexing already pares down the document to only what's needed for supersearch and now this redactor isn't having an effect on things which is why we can remove it.